newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -158,6 +159,15 @@ export const allAlertMessages: Array<{
       <Trans>Project package names should not begin with com.example</Trans>
     ),
   },
+  {
+    key: 'no-username-for-slug',
+    label: (
+      <Trans>
+        A username must be defined in your profile to be able to choose a custom
+        game URL.
```

Comment:
Should this be a shorter label like "username required to choose a custom game URL"?

Reply:
"usernames are required to choose a custom game URL" to make a sentence?

Reply:
Ok!